pkg/qliksense/install_test.go

```diff
@@ -130,6 +130,7 @@ spec:
 	}
 
 	originalOperatorString := q.GetOperatorControllerString()
+	fmt.Printf("originalOperatorString:\n%v\n", originalOperatorString)
```

Comment:
Minor comment: Should we print this out? or make this a debug log?

Reply:
@ashwathishiva: that print statement stinks I agree. But it's `only` in the test file... I could maybe remove it and just improve the test a bit more to check for the potential problem of double-including resources on Windows.

Reply:
@ashwathishiva: I'm removing it. The problem with double-resource include by packer only manifested itself on Windows and that test would fail there regardless.